Runtime/Triangulator.cs

```diff
@@ -2967,60 +2908,31 @@ static double pseudoAngle(double dx, double dy)
         public readonly bool2 isfinite(double2 v) => math.isfinite(v);
         public readonly bool le(double a, double b) => a <= b;
         public readonly bool2 le(double2 a, double2 b) => a <= b;
+        public readonly double2 lerp(double2 a, double2 b, double v) => math.lerp(a, b, v);
         public readonly bool less(double a, double b) => a < b;
         public readonly double2 max(double2 v, double2 w) => math.max(v, w);
         public readonly double2 min(double2 v, double2 w) => math.min(v, w);
         public readonly double mul(double a, double b) => a * b;
+        public readonly double2 neg(double2 v) => -v;
     }
 
     internal readonly struct IntUtils : IUtils<int, int2, long>
     {
-        public readonly long ConstTBig(float v) => (long)v;
-        public readonly long EPSILON_TBIG() => 0;
-        public readonly int MaxValue() => int.MaxValue;
-        public readonly int2 MaxValue2() => int.MaxValue;
-        public readonly long MaxDistSq() => long.MaxValue;
-        public readonly int2 MinValue2() => int.MinValue;
-        public readonly int X(int2 a) => a.x;
-        public readonly int Y(int2 a) => a.y;
-        public readonly long ZeroTBig() => 0;
-        public readonly bool SupportsMeshRefinement => false;
-
-        public long cross(int2 a, int2 b) => (long)a.x * (long)b.y - (long)a.y * (long)b.x;
-
-        public readonly bool PointInsideTriangle(int2 p, int2 a, int2 b, int2 c) {
-            var check1 = cross(p - a, b - a);
-            var check2 = cross(p - b, c - b);
-            var check3 = cross(p - c, a - c);
-
-            // Allow for both clockwise and counter-clockwise triangle layouts.
-            // TODO: Is this required, or are all triangles guaranteed to be in one orientation only?
-            return (check1 >= 0 & check2 >= 0 & check3 >= 0) | (check1 <= 0 & check2 <= 0 & check3 <= 0);
-        }
-
-        public readonly int2 CircumCenter(int2 a, int2 b, int2 c) {
+        public readonly int Cast(long v) => (int)v;
+        public readonly int2 CircumCenter(int2 a, int2 b, int2 c)
+        {
             var d = b - a;
             var e = c - a;
 
-            var bl = (long)d.x*d.x + (long)d.y*d.y;
-            var cl = (long)e.x*e.x + (long)e.y*e.y;
-
-            long div = d.x * e.y - d.y * e.x;
-
-            // If the triangle is degenerate (just a line), then we should return a point at infinity,
-            // because the circumcenter is not well-defined for this case.
-            // Integers do not have infinites, but we can use the maximum value as a substitute.
-            if (div == 0) return new int2(int.MaxValue, int.MaxValue);
+            var bl = (long)d.x * d.x + (long)d.y * d.y;
+            var cl = (long)e.x * e.x + (long)e.y * e.y;
 
-            double d2 = 0.5 / (double)div;
+            var div = (long)d.x * e.y - (long)d.y * e.x;
```

Comment:
Seriously?
No, that's it. I'm just going to fork this library.

Do you know how many hours I've spent trying to figure out your code because there are no comments anywhere? And here I am adding useful and relevant comments, only for you to remove them.
I had a bunch of PRs planned for this repo, but I am not motivated at all anymore, since you clearly do not respect the time it takes to understand the code for anyone but yourself.
Sorry for sounding harsh, but I just cannot fathom your way of coding. And it has cost me too much time already.

Good luck!

Reply:
I'm very sorry, but when contributing to someone else's project, you should consider the rules and guidelines of the repository. I deliberately avoid unnecessary comments because, in my opinion, they do not add any valuable information. I'm very sorry.

Best,
Andrzej